### DISCUSSION TITLE
User management misunderstanding

### DISCUSSION THREAD

**Discussion by hugobaret** [fief-dev/fief#182] [Q&A] [answered]
Hello everyone,

I am pretty new to fief, I started today actually, and I am trying to make the self-hosted version (docker compose) works.

I managed to get to the admin interface, but now I have a question about the user management. When adding a user, where is this user stored? Because when I look into the postgreSql db, I only see in the workspace_users table, a single row which is the admin user, whereas I created 4 new users. Is there a problem with my db?

![image](https://user-images.githubusercontent.com/72015973/235888849-d5520e45-7125-4ac1-b9ff-acc3f67ac53a.png)

![image](https://user-images.githubusercontent.com/72015973/235889323-a4a1fe50-87b5-4ab7-8a7a-afabde2ce8a4.png)

Thank you.

**Comment by fief-bailiff** [comment]
Hail, @hugobaret 👋 Welcome to Fief's kingdom!

Our team will get back to you very soon to help.

In the meantime, take a minute to star our repository ⭐️

![star-fief](https://user-images.githubusercontent.com/1144727/192523552-dc2c31fa-ae94-42fa-9ab4-7d0ba51bb7b0.png)

Farewell!

**Comment by hugobaret** [comment]
![image](https://user-images.githubusercontent.com/72015973/235891499-d849c13e-04fe-4d6f-b8c3-868df7809e2c.png)

**Comment by frankie567** [comment, marked as answer]
Hi @hugobaret 👋

That's completely expected 🙂

With PostgreSQL, we leverage a feature of the DBMS called "schemas": https://www.postgresql.org/docs/current/ddl-schemas.html In a nutshell, we are able to create "sub-database". 

The main one is here to manage the different workspaces. For each workspace, we create a single schema, named after the ID of the workspace.

You can query a schema by prefixing the table name with the schema name:

```sql
SELECT * FROM "d293a0cc-b26a-47a0-ac75-41c93756de62".fief_users;
```

**Comment by hugobaret** [comment]
Hello @frankie567 and thank you for your response, it solved my problem.
